src/references/damageable_objects.md

```diff
@@ -0,0 +1,93 @@
+---
+id: damageable_reference
+name: Damageable Objects Reference
+title: Damageable Objects Reference
+tags:
+    - Reference
+---
+
+# Damageable Objects
+
+## Summary
+
+The **Damageable Object** is a **Core Object** that implements the **Damageable** interface and has properties, events, and functions for objects which can take damage and die.
+
+The **Damageable Object** makes tracking damage and destroying objects very easy. Children of a **Damageable Object** that are hit by a projectile (or another damage source), will look for an ancestor that is a **Damageable** and automatically deduct the damage from health of the **Damageable**. You have control over what happens when the health of the **Damageable Object** reaches 0.
+
+A benefit to **Damageable Objects**, is no scripting is needed to damage an object and destroy it. This can all be controlled from the custom properties, which makes it quick and easy to setup.
+
+## Use Cases
+
+There are many use cases for the **Damageable Object**. Here are a few examples of how the **Damageable Object** could be used in a game.
+
+- Breakable Objects
+
+    Some games have objects that can be broken and reveal loot for the player. For example, a Crate that can be destroyed by players. This can be done without any scripts.
+
+- Dummy
+
+    Letting players test their weapons out on a dummy that is immortal. This allows the players to see how their weapon handles, and what damage is does.
+
+- Boss Fight
+
+    A Boss that is immune until the shield has been destroyed. Using nested **Damageable Objects**, this is easy to do.
```

Comment:
Boss --> Enemy AI (to not use the thing we're defining in the definition)
immune --> immune to damage

Always cut "this is easy". Also think a more explicit explanation of how nesting damageable objects achieves this.